client/src/app/site/pages/meetings/pages/history/components/history-banner/history-banner.component.ts

```diff
@@ -10,6 +11,14 @@ import { HistoryService } from '../../services/history.service';
 export class HistoryBannerComponent {
     public constructor(private historyService: HistoryService) {}
 
+    public get historyPositionTimestamp(): string {
+        if (!this.historyService.currentHistoryPosition?.timestamp) {
+            return ``;
+        }
+
+        return unix(this.historyService.currentHistoryPosition.timestamp).local().format(`lll`);
```

Comment:
Instead of doing this, use a pipe in the template. You'll need to import the `PipesModule` in the `module.ts`.